python/ray/data/read_api.py

```diff
@@ -562,6 +562,15 @@ def read_csv(
         >>> ray.data.read_csv( # doctest: +SKIP
         ...     ["s3://bucket/path1", "s3://bucket/path2"])
 
+        >>> # Read files that use a different delimiter. The partition_filter=None is needed here
+        >>> # because by default read_csv only reads .csv files.
+        >>> from pyarrow import csv
+        >>> parse_options = csv.ParseOptions(delimiter="\\t")
```

Comment:
it should be `delimiter="\t"` right? I tested it out in https://github.com/ray-project/ray/issues/27738#issuecomment-1211155880 .

Reply:
I tried that before and then it didn't render correctly in the docs, let me try again :)

Reply:
Ok, so the right solution that makes this work in all cases (both raw docstring and readthedocs) seems to be to put an `r` before the doc string and not escape this, which I did now :)